gems/safemode/CVE-2016-3693.yml

```diff
@@ -0,0 +1,14 @@
+---
+gem: safemode
+cve: 2016-3693
+title: Information leakage through templates
+date: 2016-04-20
+url: http://seclists.org/oss-sec/2016/q2/119
+description: |
+  A provisioning template containing `inspect` will expose sensitive
```

Comment:
This is all Foreman-specific, but the gem has nothing to do with foreman. Can you rewrite this to be specific to what was actually vulnerable for `safemode`?